fw/t/unit/test_http1_parser.c

```diff
@@ -2833,15 +2833,18 @@ TEST(http1_parser, etag)
 	EXPECT_BLOCK_RESP(RESP_ETAG_START header "\r\n" RESP_ETAG_END)
 
 	FOR_ETAG("ETag:   \"dummy\"  ",  "dummy\"", EXPECT_FALSE);
+	FOR_ETAG("ETag:   dummy  ", "dummy\"", EXPECT_FALSE);
+	FOR_ETAG("Etag: 'dummy'", "'dummy'\"", EXPECT_FALSE);
+	FOR_ETAG("Etag:   *", "*\"", EXPECT_FALSE);
```

Comment:
Shouldn't this be
```suggestion
        FOR_ETAG("Etag:   *\"", "*", EXPECT_FALSE);
```
or
```suggestion
        ETAG_BLOCK("Etag:   *\"");
```
, i.e. we pass `Etag: *"` to the parser and expect that it doesn't match to `*` (i.e. we don't allow single quote)?

Reply:
Fixed to ETAG_BLOK